views/js/runner/plugins/tools/hider/hider.js

```diff
@@ -0,0 +1,88 @@
+/**
+ * Hider Test Runner Plugin
+ *
+ * Adds a button to the Toolbar which hides the ContentArea to prevents others to see the test taker's test
+ */
+define([
+    'i18n',
+    'taoTests/runner/plugin',
+    'taoQtiTest/runner/plugins/tools/hider/contentMask',
+], function (__, pluginFactory, contentMaskFactory) {
+    'use strict';
+
+    /**
+     * Returns the configured plugin
+     */
+    return pluginFactory({
+        name: 'hider',
+
+        /**
+         * Initializes the plugin
+         */
+        init : function init() {
+            var self = this;
+            var areaBroker = this.getAreaBroker();
+            var contentArea = areaBroker.getContentArea();
+            var testRunner = this.getTestRunner();
+
+            this.button = areaBroker.getToolbox().createEntry({
+                control: 'hider',
+                text: 'Hider',
+                title: 'Hide the item',
+                icon: 'eye-slash',
+            });
+
+            testRunner
+                .on('enabletools renderitem', function() {
+                    self.enable();
+                })
+                .on('disabletools unloaditem', function() {
+                    self.disable();
+                })
+                .on('renderitem', function() {
+                    self.contentMask = contentMaskFactory(areaBroker.getContentArea(), {
+                        content: __('Please, do not check my answers! Thanks :)'),
+                    });
+                })
+            ;
+
+            this.button.on('click', function() {
+                self.contentMask.trigger('toggle');
+            });
+
+            contentArea.on('click', function() {
```

Comment:
This listener should lie on the `contentMask` itself. Otherwise you should remove the listener on each item change.